portfolio_aboutme.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <title>Jake Rooms - About me</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
+</head>
+<body>
+	<!--Html for the top navigation bar-->
+	<header class='link_bar'>
```

Comment:
Maybe use the nav tag so it is more semantic?

Reply:
I thought it was ok as there is only one link in it and it's not a nav bar. good shout though, will update if extra links are added to the top. :)

Reply:
changed it and added a class to the projects nav box